lib/views/help/requesting.es.html.erb

```diff
@@ -383,10 +383,7 @@
         </li>
       </ol>
       <p>
-        En estos casos puedes utilizar los recursos legales planteados en el punto 19.
-      </p>
-      <p>
-        Tendrás la posibilidad de categorizar tu respuesta según una lista de categorías (ver punto 17). Si no sabes cómo hacerlo te ayudaremos. <i>derechoapreguntar.org</i>
+        En estos casos puedes utilizar los recursos legales planteados en esta web: <a href="#recursos">Los Recursos en Nicaragua</a>
```

Comment:
Not sure where the URL here is meant to go. If its to an anchor tag on one of the help sections it needs to use the English name.